packages/react-voting-app/src/services/api.ts

```diff
@@ -1,30 +1,39 @@
 import axios from 'axios';
 import { hashId } from '@bvotal/common';
-import { generateMasterKeys, PassPhraseGenerator, encryptAES } from '@burstjs/crypto';
-const eaBaseUrl = `http://localhost:3000/api`;
+import { generateMasterKeys, PassPhraseGenerator, encryptAES, decryptAES } from '@burstjs/crypto';
+const eaBaseUrl = `http://localhost:3001/api`;
 
 const eaApi = axios.create({
     baseURL: eaBaseUrl,
 });
 
+const StorageKeys = {
+    Passphrase: 'p'
+}
+
 const api = {
-    getMasterKeys: (passphraseArray: string[]) => {
-        const passphrase = passphraseArray.join(' ');
+    getMasterKeys: (passphrase: string) => {
         return generateMasterKeys(passphrase);
     },
-    getRandom: () => {
-        const typedArray = new Uint32Array();
-        return window.crypto.getRandomValues(typedArray).join();
-    },
-    getPassphrase: async (hashId: string) => {
-        const random = api.getRandom();
+    generatePassphrase: async (hashId: string) => {
+        const typedArray = new Uint8Array(64);
+        const random = window.crypto.getRandomValues(typedArray).join();
```

Comment:
this turns an [1,2,3] into "1,2,3" string.... not really what we want here... I'll improve this here.... it's a bit more low level